.pre-commit-config.yaml

```
@@ -32,3 +32,11 @@ repos:
     rev: "1.20.0"
     hooks:
       - id: "tmt-tests-lint"
+
+  - repo: https://github.com/packit/pre-commit-hooks
+    rev: 8db5a24e01b9f54aaa7a800f33c4b9aa619af1b9
```

They released 1.0.0 last week: https://github.com/packit/pre-commit-hooks/releases/tag/v1.0.0

```suggestion
    rev: 1.0.0
```

---

Nice! Also @jpopelka pr @TomasTomecek you fine with us using this?

---

sure